### PR TITLE
Testing only...

### PR DESCRIPTION
**Pull request by kilfoyle**
This changes "current" for the Cloud ESS docs to ms-117.

Do not merge until release day.

### CODE REVIEW

**Comment by kilfoyle** [comment]
@elasticmachine run docs-build

**Review by lcawl — DISMISSED** [review]

**Comment by kilfoyle** [comment]
@elasticmachine run docs-build

**Comment by kilfoyle** [comment]
@elasticmachine run docs-build

**Comment by kilfoyle** [comment]
Closing in favour of https://github.com/elastic/docs/pull/3146

**Comment by kilfoyle** [comment]
Reopening to test something...

**Comment by kilfoyle** [comment]
@elasticmachine update branch

File: conf.yaml [hunk 1]
```diff
@@ -83,7 +83,7 @@ variables:
   stackcurrent: &stackcurrent 8.16
   stacklive: &stacklive [ 8.16, 7.17 ]
 
-  cloudSaasCurrent: &cloudSaasCurrent ms-116
+  cloudSaasCurrent: &cloudSaasCurrent ms-117
```
Comment (kilfoyle) [inline conf.yaml:86]:
```suggestion
  cloudSaasCurrent: &cloudSaasCurrent ms-117
  
  //test
```

**Pull request closed without merging**